api/audit-trail-gw/src/services/subscription-service.ts

```diff
@@ -111,7 +114,13 @@ export class SubscriptionService {
 			}
 		}
 
-		const state = this.streamsService.exportSubscription(res.subscriber, this.password);
+		let state = this.streamsService.exportSubscription(res.subscriber, this.password);
```

Comment:
you cant use this.password for exportSubscription if it is a privatePlus channel